docs: mention the quarkus.smallrye-openapi.path config (#2938)
quarkusio/quarkus: docs/src/main/asciidoc/openapi-swaggerui-guide.adoc

          type: string
----

[NOTE]
====

> If you do not like the default endpoint location `/openapi`, you can change it with following configuration in your `application.properties`:

Reviewer: ```suggestion
If you do not like the default endpoint location `/openapi`, you can change it by adding the following configuration to your `application.properties`:
```
